examples/realtime/plot_compute_rt_average.py

```diff
@@ -55,6 +55,7 @@
         evoked = ev
     else:
         evoked += ev
+    evoked.pick_types(meg=True, eog=False)  # leave out the eog channel
```

Comment:
As discussed in https://github.com/mne-tools/mne-python/pull/2991, you have to explicitly include eog channels if you want them in the evoked dataset.